src/main/java/de/zalando/aruha/nakadi/config/NakadiConfig.java

```diff
@@ -29,6 +33,11 @@
 
     public static final MetricRegistry METRIC_REGISTRY = new MetricRegistry();
     public static final HealthCheckRegistry HEALTH_CHECK_REGISTRY = new HealthCheckRegistry();
+    public static final String ZK_BROKERS_DEFAULT = "127.0.0.1:2181";
+    public static final String KAFKA_BROKERS_DEFAULT = "127.0.0.1:9092";
```

Comment:
Because then it's yet two other configuration parameters that will never be overridden. I don't think we need this to be configurable (it's the default value if the actual value is not set). Hence, I try to keep my configuration lean. :)
Imho, if you want to change it, you can change it in the code and redeploy. :)